### ISSUE TITLE
No CV release.

### ISSUE THREAD

**Issue by zvezdochiot** [shuwang127/SRCNN_Cpp#5]
Hi @shuwang127 .

See [stb-image-srcnn](https://github.com/ImageProcessing-ElectronicPublications/stb-image-srcnn). This is your SRCNN implementation in C without OpenCV. Use [stb](https://github.com/nothings/stb.git).

Good luck.

**Comment by shuwang127** [comment]
Thank you, @zvezdochiot. I have noticed the repository. Yes, OpenCV is bloated for this application, and using the non-OpenCV version is necessary for lightweight devices. I also noticed [libsrcnn](https://github.com/rageworx/libsrcnn) also seems to be using a non-OpenCV scheme. 

I have accepted a merge from @rageworx with the updated OpenCV4 version, which I have successfully tested on my Linux machine. I will update the ReadMe.md file tomorrow to reflect the updates and mention your efforts on the non-OpenCV schema. 

Thanks.
